### PR TITLE
new package: guile 2.2.6

### PR DESCRIPTION
**Pull request by a-schaefers**
## Checklist

- [x] Latest upstream version.
- [x] I agree to maintain this package (**required**).
- [x] I agree to notify KISS if I can no longer maintain this package (**required**).

### CODE REVIEW

**Comment by dylanaraps** [comment]
We can't package another scheme? ;)

**Comment by a-schaefers** [comment]
We already have another scheme, 'gosh' ;-)

**Comment by a-schaefers** [comment]
Btw, this pkg lays the foundation for shepherd init system

File: community/guile/depends [hunk 1]
```diff
@@ -0,0 +1,5 @@
+libtool make
+ncurses make
+gmp
+libunistring
+gc
```
Comment (dylanaraps) [inline community/guile/depends]: Sort this list.

**Pull request merged**